executor/instruction/mod.rs

```diff
@@ -875,8 +875,14 @@ impl<T> Checker<T> {
                         Comparator::Greater => |a, b| a > b,
                         Comparator::LessOrEqual => |a, b| a <= b,
                         Comparator::GreaterOrEqual => |a, b| a >= b,
-                        Comparator::Like => unimplemented_feature!(ComparatorLike),
-                        Comparator::Contains => unimplemented_feature!(ComparatorContains), // |a,b| a.unwrap_string_ref().contains(b.unwrap_string_ref()),
+                        Comparator::Like => |a, b| {
+                            regex::Regex::new(b.unwrap_string_ref())
+                                .expect("Invalid regex should have been caught at compile time")
+                                .is_match(a.unwrap_string_ref())
```

Comment:
We're recompiling the regex every time we check? Could we cache the `Regex` during translation?

Reply:
Tried but it smelt wrong throughout, raising the question as to whether we want to treat `like` as a comparator between two `String` values or some different operator class.